Redesign splash page

AkashPaloju opened the pull request:
## Overview

<!--
READ ME FIRST:
Please answer *all* questions below and check off every point from the Essential Checklist!
If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
-->

1. This PR fixes or fixes part of #19214 
2. This PR does the following: Redesigns the splash page.

## Essential Checklist

Please follow the [instructions for making a code change](https://github.com/oppia/oppia/wiki/Make-a-pull-request).

- [x] I have linked the issue that this PR fixes in the "Development" section of the sidebar.
- [x] I have checked the "Files Changed" tab and confirmed that the changes are what I want to make.
- [x] I have written tests for my code.
- [x] The **PR title** starts with "Fix #bugnum: " or "Fix part of #bugnum: ...", followed by a short, clear summary of the changes.
- [x] I have assigned the correct reviewers to this PR (or will leave a comment with the phrase "@{{reviewer_username}} PTAL" if I can't assign them directly).


## Proof that changes are correct

Desktop & Mobile devices

[splash-revisions.webm](https://github.com/oppia/oppia/assets/119884665/383e5af2-c6bc-4d38-8ebf-553ec14d6136)

Hover effects
[splash-hover-effects.webm](https://github.com/oppia/oppia/assets/119884665/c418a2cf-44ab-468e-886f-3bc5a02cbb13)


Lighthouse tests in a11y

[Screencast from 23-04-24 10:45:31 PM IST.webm](https://github.com/oppia/oppia/assets/119884665/84f7ca8c-df78-460f-9969-9e74538cb63c)

RTL Layout:
[Screencast from 23-04-24 10:43:28 PM IST.webm](https://github.com/oppia/oppia/assets/119884665/91a6384a-0791-4ec1-8b2b-038b0e91edb2)

Slow Network:

[Screencast from 23-04-24 10:51:37 PM IST.webm](https://github.com/oppia/oppia/assets/119884665/e12182de-f459-4118-802b-5102a4fb8dd1)



<!--
Add before-and-after videos/screenshots of the user-facing interface (including
the browser devtools console) to demonstrate that the changes made in this PR
work correctly. Make sure the actions taken in the before and after videos are
the same. (For changes involving responsiveness or adjustment of UI elements,
this should be a video that gradually resizes the viewport from wide to narrow
and back again.) If this PR is for a developer-facing feature, provide
videos/screenshots of the developer-facing interface instead.

When you make updates to the PR, please update these videos/screenshots as well.
You can drop videos/screenshots from previous versions of the PR.

The above should be done for all PRs, including short ones (e.g. a single-line change).
However, if the changes in your PRs are autogenerated via a script and you cannot
provide proof for the changes then please leave a comment "No proof of changes
needed because {{Reason}}" and remove all the sections below.
-->



<!--
Make sure to properly verify that everything works correctly, and that there are
no weird UI mistakes or other problems. Also, if there are any newly added fields,
try to fill them out and test that different inputs are correctly accepted/rejected.

Throttle the network (to 3G) using the browser Developer Tools (see references below).
There should be no performance or UI issues while the network is slow.

References:
 - Chrome: https://css-tricks.com/throttling-the-network/
 - Firefox: https://developer.mozilla.org/en-US/docs/Tools/Network_Monitor/Throttling
-->



<!--
In some cases this is not needed (e.g. for pages that we do not expect to
support mobile phones, or for backend-only features).

Feel free to use the Developer Tools emulator for this.

References:
 - Chrome: https://developer.chrome.com/docs/devtools/device-mode/
 - Firefox: https://firefox-source-docs.mozilla.org/devtools-user/index.html#responsive-design-mode
-->



<!--
If the PR changes the UI, make sure to add screenshots with the site
language set to Arabic as well (we use Arabic as it is a language written from right to left).
-->

## PR Pointers

- Never force push! If you do, your PR will be closed.
- To reply to reviewers, follow these instructions: https://github.com/oppia/oppia/wiki/Make-a-pull-request#step-5-address-review-comments-until-all-reviewers-approve
- Some e2e tests are flaky, and can fail for reasons unrelated to your PR. We are working on fixing this, but in the meantime, if you need to restart the tests, please check the ["If your build fails" wiki page](https://github.com/oppia/oppia/wiki/If-CI-checks-fail-on-your-PR).
- See the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia's-code-owners-and-checks-to-be-carried-out-by-developers) for what code owners will expect.

AkashPaloju committed:
Add initial splash page

AkashPaloju commented:
I will assign the reviewers once all tests are passed. Thanks!

AkashPaloju committed:
Add i18n support

AkashPaloju committed:
Add some a11y support

AkashPaloju commented:
Current Status: Waiting for reply on this [query](https://github.com/oppia/oppia/issues/19214#issuecomment-2060371606) before the 'a11y' commit

seanlip commented:
@AkashPaloju I've replied to the query. Are you planning to finish this PR?

Please also note that your CI checks are failing.

seanlip commented:
Also, one note -- where possible, try to optimize the image sizes, especially for the larger files. Thanks!

AkashPaloju commented:
Time is taking for aligining these images with the curves in all screen dimensions correctly because these complex shapes are not directly available from figma, I got to crop some of them, for aligning them correctly, working on it. 

![image](https://github.com/oppia/oppia/assets/119884665/f580b1fb-efbb-4190-b103-d2ccdbc1a517)
![image](https://github.com/oppia/oppia/assets/119884665/2d5500e7-0baa-4d70-91c3-4ee8236c1c78)

AkashPaloju committed:
fix failing tests, add a11y support

AkashPaloju committed:
Fix failing frontend and lighthouse tests

AkashPaloju committed:
Merge branch 'develop' of https://github.com/oppia/oppia into splash-page

AkashPaloju committed:
Fix failing backend and frontend coverage tests

AkashPaloju committed:
Fix prettier formatter test

seanlip commented:
Thanks @AkashPaloju, I am also adding the design leads for a review of the video!

S4v8n commented:
Hi, 
Great work so far!  
Here are some revisions for the splash page. 

Web version: 
1. "Learn through story.." title is missing white background for all three versions (see figma file for reference.)
2. Set 295px widths for all subtext under title "Fun free", "Learn by yourself" , " Diversity of subjects"
3. set 289px widths for body text "Oppia's classroom" and "Community lessons"
4. Community lessons title and button should be Dark Orange color #B25239 (for all three versions -web, mobile, tablet)
5. set 682px width for body text - "Have your own account"
6. Set 478px width body text - "For parents"
7. set 456px width body text - "For volunteers"

Mobile version: 
1. missing white background "Learn through story..."
2. CTA buttons not centered especially "Start learning" and "Or create an account"
3. Double check fonts are Capriola font, size 16 for title, Roboto font regular size 13 for body text -  "Fun free", Learn by yourself", "Diversity of subjects" Also make sure spacing between title and body text is 8px, it currently too much space.

Tablet version: 
1. missing white background "Learn through story..."
2. Set 543px width for body text - "Have your own account"
3. set 467px width for body text - "For teachers"
4. set 415px width for body text - "For parents"
5. set 447px width for body text - "For volunteers" 

Thank you for your great work!

rflore commented:
Hi Team,

**For Web:**
Header, Body Text, & CTA: "Learn through story based lessons"; The left margin is 89px, while the others are at 88px. Please update.
Header: "Have your own account"; The video shows a smaller left margin, please update to 88px.

Kurin has done a great job capturing most of the changes needed, especially the mobile CTA alignments issues. 

Nice work all!

seanlip commented:
Adding @nikitaevg for lace-frontend-reviewers.

nikitaevg commented on core/templates/pages/splash-page/splash-page.component.html:
```diff
@@ -1,147 +1,159 @@
+                 alt="Illustration of a young person in front of part of Oppia fraction lesson">
+          </picture>
+        </div>
+        <div class="oppia-splash-section-one-top-left">
```
I understand it was like this before, but now that you're redesigning it, maybe let's add meaningful names to the classes? "section-one" doesn't say much about semantics

nikitaevg commented on core/templates/pages/splash-page/splash-page.component.ts:
```diff
@@ -77,91 +68,60 @@ export class SplashPageComponent implements OnInit {
+  private _navigateToAccountCreationPage(): void {
+    this.userService.getLoginUrlAsync().then(loginUrl => {
+      if (loginUrl) {
+        setTimeout(() => {
```
Could you please explain why adding timeout here?

AkashPaloju replied:
This timeout thing is there at few places in the codebase where we navigate to Google 'sign-in' page.

For Eg [here](https://github.com/search?q=repo%3Aoppia%2Foppia%20this.userService.getLoginUrlAsync().then(loginUrl%20%3D%3E%20%7B&type=code)

I guess, this 0.15 second delay is for waiting for Google 'Sign in' to load fully

nikitaevg commented on core/templates/pages/splash-page/splash-page.component.ts:
```diff
@@ -77,91 +68,60 @@ export class SplashPageComponent implements OnInit {
+          this.windowRef.nativeWindow.location.href = loginUrl;
+        }, 150);
+      } else {
+        this.windowRef.nativeWindow.location.reload();
```
And why reload here?

AkashPaloju replied:
This reload thing is there throughout the codebase wherever we navigate to Google 'sign-in' page.

For Eg [here](https://github.com/search?q=repo%3Aoppia%2Foppia%20this.userService.getLoginUrlAsync().then(loginUrl%20%3D%3E%20%7B&type=code )

nikitaevg reviewed (changes requested):
Looks good, a couple minor comments

AkashPaloju commented:
> Hi, Great work so far! Here are some revisions for the splash page.
> 
> Web version:
> 
> 1. "Learn through story.." title is missing white background for all three versions (see figma file for reference.)
> 2. Set 295px widths for all subtext under title "Fun free", "Learn by yourself" , " Diversity of subjects"
> 3. set 289px widths for body text "Oppia's classroom" and "Community lessons"
> 4. Community lessons title and button should be Dark Orange color #B25239 (for all three versions -web, mobile, tablet)
> 5. set 682px width for body text - "Have your own account"
> 6. Set 478px width body text - "For parents"
> 7. set 456px width body text - "For volunteers"
> 
> Mobile version:
> 
> 1. missing white background "Learn through story..."
> 2. CTA buttons not centered especially "Start learning" and "Or create an account"
> 3. Double check fonts are Capriola font, size 16 for title, Roboto font regular size 13 for body text -  "Fun free", Learn by yourself", "Diversity of subjects" Also make sure spacing between title and body text is 8px, it currently too much space.
> 
> Tablet version:
> 
> 1. missing white background "Learn through story..."
> 2. Set 543px width for body text - "Have your own account"
> 3. set 467px width for body text - "For teachers"
> 4. set 415px width for body text - "For parents"
> 5. set 447px width for body text - "For volunteers"
> 
> Thank you for your great work!

Hi, @S4v8n , For the 4th point about the Dark Orange color #B25239 please refer this [query](https://github.com/oppia/oppia/issues/19214#issuecomment-2060371606)

Thanks!

AkashPaloju committed:
Address review comments 1

AkashPaloju committed:
fix lint issues

AkashPaloju commented:
Hi, @S4v8n , Thanks for your feedback, I have addressed the comments Could you PTAL at the updated videos ? 
Thanks!

seanlip commented:
@AkashPaloju Please reply to **all** the comments by @nikitaevg, thanks.

S4v8n commented:
> Hi, @S4v8n, @rflore , Thanks for your feedback, I have addressed the comments Could you PTAL at the updated videos ? Thanks!

Hi Akash, thank you for the latest video update.  The desktop version looks better.  There are couple things I noticed. 

In Desktop version: 

1. The first line "Learn anything through story based-lessons!" title should be across the page, font size: 28

2. Also are you able to apply the white background to have the curved shapes?  see image below. If you are able to do it, there are two other areas that have the curve shapes on page.  this would need to be applied to all screen versions. 

<img width="388" alt="image" src="https://github.com/oppia/oppia/assets/140122208/00c9fc3e-5fb1-4634-8ab5-cdb9b85e3313">



One small thing I notice in the Mobile portrait mode.  

- in section that starts with " Learn more about Oppia", the subtext font looks bigger than the title font. 
- >The titles "Learn by yourself" and the all titles should be in Capriola regular font, font size: 16, the subtext should be Roboto regular font, font size: 13.  

Thank you again for your work. It is looking great. 

Best,
kurin

AkashPaloju commented:
> > Hi, @S4v8n, @rflore , Thanks for your feedback, I have addressed the comments Could you PTAL at the updated videos ? Thanks!
> 
> Hi Akash, thank you for the latest video update. The desktop version looks better. There are couple things I noticed.
> 
> In Desktop version:
> 
> 1. The first line "Learn anything through story based-lessons!" title should be across the page, font size: 28
> 2. Also are you able to apply the white background to have the curved shapes?  see image below. If you are able to do it, there are two other areas that have the curve shapes on page.  this would need to be applied to all screen versions.
> 
> <img alt="image" width="388" src="https://private-user-images.githubusercontent.com/140122208/331687406-00c9fc3e-5fb1-4634-8ab5-cdb9b85e3313.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU5NjkxMjcsIm5iZiI6MTcxNTk2ODgyNywicGF0aCI6Ii8xNDAxMjIyMDgvMzMxNjg3NDA2LTAwYzlmYzNlLTVmYjEtNDYzNC04YWI1LWNkYjliODVlMzMxMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxN1QxODAwMjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNTJmYTczNjYxNGE4OWJkNzM2OTM4YjJhNGI2NzIxYTQ1M2FiMjVmNDdiYjI3M2I1NDExMTIzY2VhZDQxODE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.gj2XDaucrmqL9S3V9ITC2Ioo3cUZPGhnVvAY9uP1LnM">
> One small thing I notice in the Mobile portrait mode.
> 
> * in section that starts with " Learn more about Oppia", the subtext font looks bigger than the title font.
> * > The titles "Learn by yourself" and the all titles should be in Capriola regular font, font size: 16, the subtext should be Roboto regular font, font size: 13.
> 
> Thank you again for your work. It is looking great.
> 
> Best, kurin

Thanks  @S4v8n ,  I will do the remaing changes but I won't be to do the white background thing for the curved shapes given my ongoing GSoC period with Oppia.

S4v8n commented:
> > > Hi, @S4v8n, @rflore , Thanks for your feedback, I have addressed the comments Could you PTAL at the updated videos ? Thanks!
> > 
> > 
> > Hi Akash, thank you for the latest video update. The desktop version looks better. There are couple things I noticed.
> > In Desktop version:
> > 
> > 1. The first line "Learn anything through story based-lessons!" title should be across the page, font size: 28
> > 2. Also are you able to apply the white background to have the curved shapes?  see image below. If you are able to do it, there are two other areas that have the curve shapes on page.  this would need to be applied to all screen versions.
> > 
> > <img alt="image" width="388" src="https://private-user-images.githubusercontent.com/140122208/331687406-00c9fc3e-5fb1-4634-8ab5-cdb9b85e3313.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU5NjkxMjcsIm5iZiI6MTcxNTk2ODgyNywicGF0aCI6Ii8xNDAxMjIyMDgvMzMxNjg3NDA2LTAwYzlmYzNlLTVmYjEtNDYzNC04YWI1LWNkYjliODVlMzMxMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxN1QxODAwMjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNTJmYTczNjYxNGE4OWJkNzM2OTM4YjJhNGI2NzIxYTQ1M2FiMjVmNDdiYjI3M2I1NDExMTIzY2VhZDQxODE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.gj2XDaucrmqL9S3V9ITC2Ioo3cUZPGhnVvAY9uP1LnM">
> > One small thing I notice in the Mobile portrait mode.
> > 
> > * in section that starts with " Learn more about Oppia", the subtext font looks bigger than the title font.
> > * > The titles "Learn by yourself" and the all titles should be in Capriola regular font, font size: 16, the subtext should be Roboto regular font, font size: 13.
> > 
> > Thank you again for your work. It is looking great.
> > Best, kurin
> 
> Thanks @S4v8n , I will do the remaing changes but I won't be to do the white background thing for the curved shapes given my ongoing GSoC period with Oppia.

Sounds good @AkashPaloju, Thanks for the response.

nikitaevg reviewed (approved):
Thanks!

stepmalt commented:
The time stamps here are for the first video on this page:

Learn anything, anywhere with Oppia—your free educational platform.

At 0:07 — having your *own account. And add a line break before Through our Learner Groups.

0:10: Oppia is a non-profit project built by volunteers from around the world. We always need more people to create, improve and translate lessons, contribute graphics, and more!

Also only capitalize Subscribe in Subscribe to our newsletter at the bottom.

Since we are moving away from title case, don’t capitalize Teachers or Parents in the header for “For Teachers” and “For Parents”

 Also, there’s not consistency with these buttons. Are we doing title case in buttons? I thought we weren’t, but I’m unsure. @seanlip

AkashPaloju commented:
> > Hi, @S4v8n, @rflore , Thanks for your feedback, I have addressed the comments Could you PTAL at the updated videos ? Thanks!
> 
> Hi Akash, thank you for the latest video update. The desktop version looks better. There are couple things I noticed.
> 
> In Desktop version:
> 
> 1. The first line "Learn anything through story based-lessons!" title should be across the page, font size: 28
> 2. Also are you able to apply the white background to have the curved shapes?  see image below. If you are able to do it, there are two other areas that have the curve shapes on page.  this would need to be applied to all screen versions.
> 
> <img alt="image" width="388" src="https://private-user-images.githubusercontent.com/140122208/331687406-00c9fc3e-5fb1-4634-8ab5-cdb9b85e3313.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYyODM3MjAsIm5iZiI6MTcxNjI4MzQyMCwicGF0aCI6Ii8xNDAxMjIyMDgvMzMxNjg3NDA2LTAwYzlmYzNlLTVmYjEtNDYzNC04YWI1LWNkYjliODVlMzMxMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMVQwOTIzNDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMGJhYTJiM2MxZWIwZDVjMTVlNjE2Yzk1MTAyNWI1YzA4YTMxYzlhNzczMjdmY2RlY2JmNGU0NTQ1ZjRlZjk4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ez8umyKpeZmv6SHz-XcxUxYbBDeyb120n8XhlOLxUQs">
> One small thing I notice in the Mobile portrait mode.
> 
> * in section that starts with " Learn more about Oppia", the subtext font looks bigger than the title font.
> * > The titles "Learn by yourself" and the all titles should be in Capriola regular font, font size: 16, the subtext should be Roboto regular font, font size: 13.
> 
> Thank you again for your work. It is looking great.
> 
> Best, kurin

Hi @S4v8n , 
Reply for 'The first line "Learn anything through story based-lessons!" title should be across the page, font size: 28' 
 In this [figma file](https://www.figma.com/design/YWe7SqfUVjxlJLKTUn0UZa/Project-2?node-id=6727-476954&t=yGe0Vbvid1MWk5CZ-0) given in the issue thread, in web version, the title is not across the page - fontsize-32px, the styles you are saying are for the tablet landscape mode. Also the text is 'Learn through story 
based-lessons!'. So should I remain with that or change them as you said, Thanks!

AkashPaloju commented:
@S4v8n , @stepmalt , Could you PTAl at the updated videos. Thanks!

AkashPaloju committed:
Address design comments 2

S4v8n commented:
> @S4v8n , @stepmalt , Could you PTAL at the updated videos. Thanks!

Hi Akash, 

Thank you for the latest changes.  Here are my comments. These are smaller things I noticed that more spacing related. 

Web: 
1. you do not need to change the text format for "Learn through story based..." , please keep it how you have currently have it. 
2. In Oppia classroom and Community lessons: spacing between graphic image ->subtext -> button, should be 24px. Please verify spacing. 
3. "Having your account.." the subtext has a single word on the last line, please adjust to have it on the 2nd line.  As a rule we should not have single word on a line, it must be two or more. (there will other areas that need to be updated as well)
4. "For teachers"  - 24px spacing between graphics / subtext/ button
5. "For volunteers" - single word "more!" needs to be adjust to the 3rd line. Spacing 24px graphics/subtext/ button

Mobile
6. "What does Oppia have to offer.." - Verify the text is Capriola , weight 400, 20px, and separated into two lines. it currently looks smaller
7. Oppia classroom - spacing 16px - between graphics/subtext/ button
8. Community lessons - move "50+" to be with languages on the 3rd line.  Spacing 16px -graphics/text/ button
9. For parents - "feature" need to be on the 4th line
10. For volunteer - text need to be below the Oppia graphic and above the button

Tablet
11. "anywhere" need to be on 2nd line.
12. Oppia classroom and Community lessons - spacing 24px - graphics/ subtext/ button
13. For teachers, "Through our learners sentence, should not have a break, it should continue on the same line a previous sentence.  spacing 24px -graphics/ subtext/ button
14. For parents, "feature" need to be on the 3rd line with sentence, spacing 24px 
15. For volunteers, spacing 24px

Thank you again for your work, it is looking really good. 

Best,
Kurin

AkashPaloju commented:
@S4v8n , I have addressed your comments, PTAL at the updated video. Thanks!

S4v8n commented:
> @S4v8n , I have addressed your comments and verified things you said, PTAL at the updated video. Thanks!

@AkashPaloju looks good! Thanks for the quick turn around. I have no further comments.

AkashPaloju committed:
Address design comments 3

AkashPaloju committed:
Address failing tests

AkashPaloju commented:
@seanlip Reviewers and Designers have approved the changes and all tests are passed. Please add this to Merge queue. Thanks!

seanlip commented:
Hi @AkashPaloju, sorry for the last minute notes, but since this is going to get launched once it's merged I thought I'd better do a quick pass through the text. In some cases we are promising things that are not live yet. Could you please make the following changes and file an issue to revert them once the relevant functionality for learner groups and multiple classrooms is in place?

- "use to learn anything, anywhere" --> "use to learn key skills, anywhere".
- Drop "Through our Learner Groups feature you can assign lessons and follow your student's progress." under "For teachers". 
- Drop "You can also see your child's progress through our Learner Group feature" under "For parents"

Also, some general textual updates (that can be permanent):
- "Diversity of subjects" --> "Diverse subjects"
- "lessons of multiple subjects" --> "lessons in multiple subjects"
- "volunteers all over the world in 50+ languages" --> "volunteers all over the world"
- Having your own account --> "Have your own account"
- "having your account" --> "creating your account"
- Drop "ad-free" from "For parents"

Also @stepmalt, re consistency of button text, that's up to the UXW team. Could you please confirm what the current style guide recommendation is?

Finally @AkashPaloju Can you put these changes behind a feature flag so we can do A/B experiments on the splash pages? I think that would help us gain confidence in the new page before fully launching it.

Thanks! (And sorry for the late response.)

seanlip commented on core/templates/services/site-analytics.service.ts:
```diff
@@ -137,6 +137,30 @@ export class SiteAnalyticsService {
   }
 
+  registerClickCreateAccountButtonEvent(): void {
+    this._sendEventToLegacyGoogleAnalytics(
```
@AkashPaloju I don't think legacy GA works any more, we should remove it. Let's use the non-legacy version of this instead.

Also could you please add another parameter along with the events common to both pages that allows us to pass along the value of the feature flag? Then we can roll this out at 50% and see how the click rates compare between the two.

seanlip commented on core/templates/pages/splash-page/splash-page.component.html:
```diff
@@ -1,147 +1,171 @@
-              [innerHTML]="'I18N_SPLASH_ICON_ONE_TEXT' | translate">
-        </span>
-      </div>
+  <div class="oppia-splash-features-section" dir="auto" role="region" >
```
Hi @AkashPaloju -- would it be possible to make a splash-page-one.component.html and a splash-page-two.component.html, then use a feature flag value to ng-if between those? That way we can get some validation of this page before we launch it fully, so that we can make sure it is better than the current one.

We can keep that running for a couple months and then analyze the results at the end to decide which version to go with finally. OK for both these pages to use the same JS component if that makes things easier, and OK for these pages to contain duplicate code if needed since we're going to drop one anyway.

seanlip reviewed (changes requested):
Thanks @AkashPaloju for the PR, sorry for my late reply on it. 

I was thinking more about the launch for this and I realized that it would probably be good to ensure that the design actually works well, so I suggest that we do a 50% feature rollout for this since we now have the functionality to do so.

(Note that for the other static pages you're doing though, I think we can definitely just go ahead with those changes since the current pages have clearer deficiencies. But for the splash page I think it is worth doing this since we still have traffic from the current page, and also this -- i.e. A/B tests -- is the sort of thing we should probably get some experience doing. PTAL at the changes suggested and let me know if you think that any of them would be difficult to do, I know the system reasonably well and can probably advise. Thanks!)

seanlip commented:
Yes, that's all 100% correct. Thanks for checking!
